fuzzing/blake.go

```diff
@@ -22,6 +22,7 @@ func fillBlake(gst *GstMaker, fork string) {
 		Data:       []string{randHex(100)},
 		GasPrice:   big.NewInt(0x10),
 		To:         dest.Hex(),
+		Sender:     sender,
```

Comment:
Hey, maybe we separate the `Sender` change if not contentious? It should lower the noise.

Reply:
This repo is pretty wild west, I'm fine with merging stuff that is borderline experimental, so IMO no need to split it up. I'm fine with merging this as if it's "ready for review", doesn't break CI, and is mergeable.